.github/workflows/build.yml

```diff
@@ -67,11 +67,7 @@ jobs:
         run: clojure -A:cljs -P
 
       - name: Fetch yarn deps
-        run: yarn install
```

Comment:
yarn has a special flag for installing dependencies in CI which will prompt errors when lock file changes. See
https://classic.yarnpkg.com/lang/en/docs/cli/install/#:~:text=If%20you%20need%20reproducible%20dependencies%2C%20which%20is%20usually%20the%20case%20with%20the%20continuous%20integration%20systems%2C%20you%20should%20pass%20%2D%2Dfrozen%2Dlockfile%20flag.

Reply:
Good to know 👍